### ISSUE TITLE
[FEATURE REQUEST] Notification badge for vertical taskbar

### ISSUE THREAD

**Issue by dxps** [marticliment/ElevenClock#1274]
### Please confirm these before moving forward

- [X] I have searched for my feature proposal and not found a work-in-progress/duplicate/resolved/discarded issue.
- [X] This proposal is a completely new feature. If you want to suggest an improvement or an enhancement, please use [this template](https://github.com/marticliment/ElevenClock/issues/new?assignees=marticliment&labels=enhancement&projects=&template=enhancement-improvement.yml&title=%5BENHANCEMENT%5D+%28Enter+your+description+here%29)

### Describe the new feature

In case of a vertical taskbar, the notification badge - although important (so I won't hide it) - makes the clock bigger than the width, as shown below.

An option to show it before or below the clock would be great.

![image](https://github.com/marticliment/ElevenClock/assets/14140226/73b71184-92ad-4768-baba-867872a233af)

Thanks!

### Describe how this new feature could help users.

This is obviously a cosmetics aspect.

**Comment by kienerj** [comment]
I agree this would be helpful together with a vertical bar like from start11 or startallback
